ansible/README.md

```diff
@@ -0,0 +1,190 @@
+# Ansible playbook for Tendermint applications
+
+![Ansible plus Tendermint](img/a_plus_t.png)
+
+* [Prerequisites](#Prerequisites)
+* [Ansible setup](#Ansible setup)
+* [Running the playbook](#Running the playbook)
+
+The playbooks in this folder run [ansible](http://www.ansible.com/) roles which:
+
+* install and configure basecoin or ethermint
+* start/stop basecoin or ethermint and reset their configuration
+
+## Prerequisites
+
+* Ansible 2.0 or higher
+* SSH key to the servers
+
+Optional for DigitalOcean droplets:
+* DigitalOcean API Token
+* python dopy package
+
+Head over to the [Terraform folder](https://github.com/tendermint/tools/tree/master/terraform-digitalocean) for a description on how to get a DigitalOcean API Token.
+
+Optional for Amazon AWS instances:
+* Amazon AWS API access key ID and secret access key.
+
+The cloud inventory scripts come from the ansible team at their [GitHub](https://github.com/ansible/ansible) page. You can get the latest version from the contrib/inventory folder.
```

Comment:
Cloud inventory scripts are dynamic scripts (usually in Python) that can query a specific cloud infrastructure for the list of nodes running. They are in the inventory folder but only the Digital Ocean script was copied over for now. If you run the script, it's going to give you a list of nodes running on Digital Ocean with your account (you have to provide the API key). Ansible can directly read this output and apply the changes on the nodes as it was an ansible inventory.